app/proguard-rules.pro

```diff
@@ -12,6 +12,7 @@
 }
 -keepclassmembers enum * {*;}
 
+-keepclassmembers enum * {*;}
```

Comment:
Why you declare this rule twice ?

Reply:
ah, seems missed that when resolving conflicts